.github/workflows/check.yml

```diff
@@ -32,8 +32,9 @@ jobs:
 
       - name: Install additional dependencies
         run: |-
-          sudo curl -fsSL https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash | sudo bash -s -- latest /bin
-
+          sudo curl -fsSL  --header "Authorization: Bearer $GITHUB_TOKEN" https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash | sudo bash -s -- latest /bin
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Nit: this shouldn't be necessary here, I don't think downloads themselves are rate limited, just the APIs that the `d2` installer uses. Ditto for the hadolint installer below.